_rules/auto-update-text-efbfc7.md

```diff
@@ -0,0 +1,390 @@
+---
+id: efbfc7
+name: Text content that updates automatically can be paused, stopped or hidden
+rule_type: atomic
+description: |
+  This rule checks that there are mechanisms to pause, stop or hide the auto-updating of text content
+accessibility_requirements: # Remove whatever is not applicable
+  wcag20:2.2.2: # Pause, Stop, Hide (A)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+input_aspects:
+  - DOM Tree
+  - CSS Styling
+acknowledgements:
+  authors:
+    - Carlos Duarte
+---
+
+## Applicability
+
+The rule applies to any [visible][] [text content][] in an [HTML document][] if:
+
+- the [text content][] is removed or replaced by new content that does not contain the previous content; and
```

Comment:
I don't think the definition of text content can be used here. Text content is computed from a node, it isn't a node in itself. What nodes should be tested here? Take this example:

```html
<div id="x"><b>Hello world</b></div>
<script>
setTimeout(() => document.querySelector('#x').innerHTML = '<i>Goodbye</i>', 1000)
</script>
```

If you mean to compute the text content of the `b` element, the text content isn't replace. The `b` element itself is. But for the `div`, the text content is replaced. You could maybe say the text content of the body, although I'm not sure how well that works. I'd prefer a clear definition of what "removed or replaced" means in that case.

Reply:
I'm not really understanding what the problem is, then. According to the applicability, the rule is not applicable to the text content of the `b` element and is applicable to the text content of the `div` element. 
From the user's perspective, there is textual content being replaced. From the rule's perspective, the applicability is catching that. 
@WilcoFiers do you still think it needs to be changed?